network-store-iidm-impl/src/test/java/com/powsybl/network/store/iidm/impl/RemoveDanglingSwitchesTopologyTest.java

```diff
@@ -45,7 +45,7 @@ public void afterRemoval(String id) {
 
     @Test
     public void testRemoveAndClean() {
-        Network network = FourSubstationsNodeBreakerFactory.create();
+        Network network = FourSubstationsNodeBreakerFactory.create(new NetworkFactoryImpl());
```

Comment:
why do you need this, there is already a `NetworkFactoryService` for network store impl in the classpath?

Reply:
Removed

Reply:
Bug fixed in CollectionCache